CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.112.2 (2020-10-07)
```

Comment:
Would this be a patch or minor version bump?

Reply:
Minor version bump, new feature.